### ISSUE TITLE
sensorsADC does not log to file

### ISSUE THREAD

**Issue by akobelan** [inertialsense/inertial-sense-sdk#45]
Platform: Linux 5.4.51-v7l+ #1333 SMP
Hardware: Raspberry Pi 4
gcc version 8.3.0 (Raspbian 8.3.0-6+rpi1) 

`./cltool -c=/dev/serial0 -msgSensorsADC -s -lon -lt=csv`

will produce an empty file called sensorsLSB.csv ... and not sensorsADC.csv with only the header:

_ID_

After some significant edits, I can coax the data to appear as, (follows), using "-s".

`DID_SENSORS_ADC: time 243351.691, bar 360176.00, barTemp 531648.00, humidity 31508.00, ana[0.000, 0.000, 0.000, 0.000], mpu[0]: pqr[5, 2, -8], acc[13, -13, -2103], mag[159, -43, 437], temp 1246.517mpu[1]: pqr[10, 0, -22], acc[26, -23, -2124], mag[380, -54, 448], temp 2218.354
`

Pretty display gives:

```

$ Inertial Sense.  Connected.  Press CTRL-C to terminate.  Rx 127

DID_DEV_INFO: SN32494, Fw 1.8.2.0 r4, 2020-07-30 16:49:57, Proto 1.2.96.8
DID_SENSORS_ADC: time 243426.819, bar 360336.00, barTemp 532288.00, humidity 31423.00, ana[0.000, 0.000, 0.000, 0.000], 
	mpu[0]: pqr[    1,     1,    -7], acc[   10,   -17, -2090], mag[  161,   -49,   437], temp 1270.923
	mpu[1]: pqr[    9,     0,   -17], acc[   21,   -21, -2121], mag[  385,   -55,   459], temp 2326.080


```

I can't seem to generate data in the logged file.

**Comment by akobelan** [comment]
I'm interested in getting the freshly converted analog voltages to digital as close to the source as possible.

**Comment by akobelan** [comment]
I figured this out. SensorsADC was classified as a "USE_IS_INTERNAL" DID. There seems to have been an effort to expose this DID for the general population. 

I had to modify several lines of code in ISDisplay.cpp, ISDisplay.h, and ISDataMappings.cpp. I'm sure this will be fixed in an upcoming release.  While I'll never get back the several hours spent tracking this down, I learned quite a bit about the backend software.
